README.md

```diff
@@ -153,13 +156,16 @@ Do this if you want to use the remote server for building and deploying.
     Where the `$user` shell variable contains the name of your shell account on the remote machine.
 
 
-### Uploading secrets and access to deploy on specific gcloud environment
-This is a separate step that is needed on the new server and in the case of changes in secrets
+### Access to deploy on specific gcloud environment
+This is a separate step that is needed for each linux user on the server that want to access to deploy on the clusters.
+The `$user_name` variable controls which user gets permission for deploying process.
+This step require knowledge of cloud password to begin decrypting process.
```

Comment:
Please reword:
```
### Authorizing a user account on the build server to access the clusters
This step must be performed separately for every user of the build server who needs to be able to access other parts of the project infrastructure on Google Cloud with `kubectl` or `gcloud`.
It can be performed by user himself or an admin who can impersonate him with `sudo`.

The `$user_name` variable below indicates the user account to be authorized.
To perform this step you need to have the .vault files with encrypted secrets in your local `concent-secrets/` directory.
Only cloud secrets are required in this case.
Ansible will prompt you for password required to decrypt them.
```